handlers/main.yml

```diff
@@ -4,3 +4,7 @@
     name: etcd
     daemon_reload: yes
     state: restarted
+    delegate_to: "{{ item }}"
```

Comment:
`unsupported parameter for module: delegate_to`
This line has wrong indentation